src/libraries/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj

```diff
@@ -6,9 +6,10 @@
     <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  <PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0' ">
```

Comment:
I should've caught this sooner. I'm wondering if this should be NETCOREAPPCURRENT instead of netcoreapp3.0?

Reply:
In this case, no. If you take a look at the particular proj file, it is hardcoding `netcoreapp3.0` in most places.

Reply:
Oh I see, you just refactored this. Still, the `USE_INTERNAL_ACCESSIBILITY` flag itself should be turned on only for NETCOREAPPCURRENT maybe?

Reply:
I think you can test this by trying both `-f netcoreapp30` and `-f netcoreapp50` locally

Reply:
Nope, it's not needed. If you look at the item group at line 36, you'll see that the shims are only included for `netcoreapp3.0`.